src/containers/AccountConnection.jsx

```diff
@@ -186,34 +195,67 @@ class AccountConnection extends Component {
             src={this.getIcon(connector)} />
         </div>
         <div className={styles['col-account-connection-content']}>
-          <div className={styles['col-account-connection-form']}>
-            { account
-              ? <h4>{t('account.connection.account.title')}</h4>
-              : <div>
-                <h3>{t('account.connection.title', { name: connector.name })}</h3>
-                <p className={styles['col-account-connection-security']}>
-                  <svg>
-                    <use xlinkHref={securityIcon} />
-                  </svg>
-                  {t('account.connection.security')}
+          { jobInBackground
+            ? <div className={styles['col-account-connection-form', 'col-account-timeout']}>
+                <h3>{t('account.connected.title', { name: connector.name })}</h3>
+                <p>{t('account.connected.description', { name: connector.name })}</p>
+                <p>
+                  {t('account.connected.ongoingSync', { name: connector.name })}
+                  <br />
+                  <span className={styles['bills-folder']}>{account.folderId}</span>
                 </p>
+                <p><a href={`/apps/files/#folders/${account.folderId}`} className={styles['link-to-folder']}>{t('account.connected.textLinkToBillsFolder')}</a></p>
```

Comment:
As we removed this feature from the scope of this PR, it's better to totally remove this link until it's ready.